assets/js/modules/analytics/datastore/notifications.js

```diff
@@ -37,6 +38,13 @@ export const selectors = {
 	 * @return {Object} The service description, learn more label and URL.
 	 */
 	getSetupSuccessContent: createRegistrySelector( ( select ) => () => {
+		const isGA4DashboardView =
```

Comment:
Since we're switching the sites going through this new setup flow to the GA4 dashboard view automatically, there is no point in showing the `You’ll only see Universal Analytics data for now` message.